app/i18n/nl/conf.php

```diff
@@ -178,7 +178,7 @@
 			'view' => 'Weergave',
 		),
 		'hide_read_feeds' => 'Categorieën en feeds zonder ongelezen artikelen verbergen (werkt niet met “Toon alle artikelen” configuratie)',
-		'img_with_lazyload' => 'Gebruik "lazy load" methode om afbeeldingen te laden',
+		'img_with_lazyload' => 'Gebruik “lazy load” methode om afbeeldingen te laden',
```

Comment:
In Dutch we'd traditionally use:
```suggestion
		'img_with_lazyload' => 'Gebruik „lazy load” methode om afbeeldingen te laden',
```

Though for this particular example I'm not sure if quotation marks make all that much sense anyway:
```suggestion
		'img_with_lazyload' => 'Gebruik <em>lazy load</em> methode om afbeeldingen te laden',
```

Reply:
As you prefer :-)